src/shared/links/PrimaryButtonLink.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+export const PrimaryButtonLink = styled(Link)`
```

Comment:
we won't need this link class low level component just remove and user btn-link class of bootstrap

Reply:
Removed it as asked